src/drivers/telemetry/frsky_telemetry/sPort_data.cpp

```diff
@@ -333,7 +333,7 @@ void sPort_send_GPS_CRS(int uart)
 	/* send course */
 
 	/* convert to 30 bit signed magnitude degrees*6E5 with MSb = 1 and bit 30=sign */
-	int32_t iYaw = s_port_subscription_data->local_pos.yaw * 18000.0f / M_PI_F;
+	int32_t iYaw = matrix::Eulerf(matrix::Quatf(s_port_subscription_data->odometry.q)).psi() * 18000.0f / M_PI_F;
```

Comment:
What are you trying here? Wrapping around yaw?

Reply:
This is actually to get yaw from the `vehicle_local_position`, since now it has the full quaternion instead of just the yaw field.